src/metasploit/msfrpc.py

```diff
@@ -208,7 +208,8 @@ def __init__(self, password, **kwargs):
                 self.client = HTTPSConnection(self.server, self.port, context=ssl._create_unverified_context())
         else:
             self.client = HTTPConnection(self.server, self.port)
-        self.login(kwargs.get('username', 'msf'), password)
+        self.username = kwargs.get('credentials').get('username', 'msf')
```

Comment:
Revert this

Reply:
@pmara-r7 , can you please drop a link here to the change in connection schema structure in the plugin you referred to?

Reply:
I think it happened before the migration to insight-connect repo so unfortunately no I cannot find it. We are reverting because we can change it at the calling site (within the plugin)